cmd/internal/distributor/distributor.go

```diff
@@ -162,7 +165,7 @@ func (d *Distributor) GetCheckpointWitness(ctx context.Context, logID, witID str
 // by both the log and the witness specified, and be larger than any previous checkpoint distributed
 // for this pair.
 func (d *Distributor) Distribute(ctx context.Context, logID, witID string, nextRaw []byte) error {
-	counterCheckpointUpdateRequests.Inc()
+	counterCheckpointUpdateRequests.WithLabelValues(witID).Inc()
```

Comment:
Best to put this line after the witness ID is verified below, or else this opens up the counter to spam/abuse.